be/src/io/fs/file_reader.h

```diff
@@ -88,6 +88,8 @@ class FileReader : public doris::ProfileCollector {
 
     virtual const std::string& get_data_dir_path() { return VIRTUAL_REMOTE_DATA_DIR; }
 
+    virtual Status update_size() { return Status::NotSupported("update_size is not supported"); }
```

Comment:
I think we can leave the default implements as empty? just return `Status::OK()`.
Because i am afraid that some new file reader may forget to implement this method and will return error.
Leave it as empty can at least remain the behavior unchanged.